### PR TITLE
fix: layout bugs on mobile

### PR DESCRIPTION
**Pull request by jimniels**
Currently:

- Command palette shows on mobile
- Permission overlay buttons wrap

We don't want either of these.

Screenshot 2024-03-28 at 3 55 14 PM

To test:

Open [a shared file on the preview branch on mobile](https://quadratic-git-fix-mobile-layout-bugs.vercel.quadratic-preview.com/file/f32db818-2fe5-4861-923d-d9871466a3ca) and see that the command palette icon is gone and the buttons don't wrap.

### CODE REVIEW

File: quadratic-client/src/ui/components/PermissionOverlay.tsx [hunk 1]
```diff
@@ -30,9 +30,9 @@ export function PermissionOverlay() {
     return (
       <FixedBottomAlert>
         <Type>
-          <strong>Welcome to Quadratic.</strong> You must log in to edit this file.
+          <strong>Welcome to Quadratic.</strong>
```
Comment (ddimaria) [inline quadratic-client/src/ui/components/PermissionOverlay.tsx:33]: Just making sure this copy deletion was intentional?

**Comment by ddimaria** [comment]
Confirmed that the command pallet is hidden and that there is no wrapping on `log in` and `sign up` in iPhone 15 (smaller format).

**Review by ddimaria — APPROVED** [review]
One non-blocking question.

**Comment by jimniels** [comment]
Yup, it was. Implied the presence of the buttons. Merging...

**Pull request merged**